ui/app/src/components/ItemMenu/ItemMenu.tsx

```diff
@@ -0,0 +1,489 @@
+/*
```

Comment:
file has pending todos

Reply:
not sure what are we going to do with the embedded items, using this menu.